README.md

```diff
@@ -278,9 +278,12 @@ Preestablishing configuration and identity is the recommended way of using this
 
 See metadata.json for supported platforms
 
-## Development
+## Contributing
```

Comment:
the corresponding link needs to be updated in the TOC at the top of this document

Reply:
Done